tests/src/Parser/DeprecatedHelperVisitor/data/deprecation-helper-test.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace GroupLegacy;
```

Comment:
This should be a more sane namespace i guess

Reply:
It's good enough for the test data :)